docs/DeveloperGuide.md

```diff
@@ -413,59 +451,63 @@ The following sequence diagram shows how `DeleteTagCommand` interacts with `Logi
 
 --------------------------------------------------------------------------------------------------------------------
 
+<div style="page-break-after: always;"></div>
+
 ## **Appendix: Requirements**
 
 ### Product scope
 
 **Target user profile**:
 
 * Financial advisors based in Singapore
-* Has a need of managing clients who have Singapore mobile phone numbers
-* Has a need of scheduling meetings with a significant number of clients
+* Has a need to manage clients who have Singapore mobile phone numbers
+* Has a need to schedule meetings with a significant number of clients
 * Prefers typing over mouse interactions and types fast
 * Is reasonably comfortable using CLI apps
 * Has English as his first language
 
-**Value proposition**: FINDvisor enables financial advisors to quickly save and retrieve the following contact’s information such as name, mobile phone number, email and physical address. It also allows financial advisors to add a remark and tags to specific contacts where needed, while offering the capability to manage their meeting details with their contacts.
+**Value proposition**: FINDvisor enables financial advisors to quickly save and retrieve the following contact’s information such as name, mobile phone number, email, and physical address. It also allows financial advisors to add a remark and tags to specific contacts where needed while offering the capability to manage their meeting details with their contacts.
```

Comment:
Since the Value Proposition is used in other parts of the codebase, I think it would be better to keep the original text since the punctuation does not really affect it.